test/component/summary/testDeceasedSection.js

```diff
@@ -37,7 +37,6 @@ describe('summary-deceased-section', () => {
                         firstName: deceasedContent.name.firstName,
                         lastName: deceasedContent.name.lastName,
                         alias: deceasedContent.alias.question.replace('{deceasedName}', deceasedContent.alias.theDeceased),
-                        dobKnown: deceasedContent.dobknown.question.replace('{deceasedName}', ''),
                         dod: deceasedContent.dod.question,
```

Comment:
why wasnt this removed?

Reply:
you mean the dod? dod is day of death so it's not removed. we only removed dobknown and dob. (they do looks so similar)

Reply:
sorry - my mistake